src/scene_battle_rpg2k3.cpp

```diff
@@ -783,14 +793,34 @@ void Scene_Battle_Rpg2k3::ProcessActions() {
 		}
 	}
 
-	if (help_window->IsVisible() && message_timer > 0) {
-		message_timer--;
-		if (message_timer <= 0)
+	if (help_window->IsVisible()) {
+		if ((state == State_SelectActor || state == State_SelectCommand || state == State_AutoBattle) && battle_action_wait == 0) {
 			help_window->SetVisible(false);
+		}
 	}
 
 	switch (state) {
 		case State_Start:
+			if (start_message_shown == 0) {
+				start_message_shown = 1;
+				if (!lcf::Data::terms.battle_start.empty()) {
+					ShowNotification(ToString(lcf::Data::terms.battle_start));
+					SetWait(10, 80);
+				}
+				return;
+			}
+			if (start_message_shown == 1) {
+				start_message_shown = 2;
+				if (!lcf::Data::terms.special_combat.empty() && (Game_Battle::GetBattleCondition() != lcf::rpg::System::BattleCondition_none || first_strike)) {
```

Comment:
`auto cond = Game_Battle::GetBattleCondition()` then use cond everywhere. Not only is this faster in the case the function is not inlined, it makes the code more compact and easier to read.

Reply:
Changed.